docs/astro/sql/operators/load_file.rst

```diff
@@ -0,0 +1,162 @@
+
+    .. literalinclude:: ../../../../example_dags/example_load_file.py
+       :language: python
+       :start-after: [load_file_example_1_start]
+       :end-before: [load_file_example_1_end]
+
+Case 2: Load into pandas dataframe
+    If you don't pass the `output_table` to the load_file operator it converts the file into a pandas dataframe and returns the reference to dataframe.
+
+    .. literalinclude:: ../../../../example_dags/example_load_file.py
+       :language: python
+       :start-after: [load_file_example_2_start]
+       :end-before: [load_file_example_2_end]
+
+Parameters to use when loading a file to the database table
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#. **ndjson_normalize_sep** - This parameter is useful when the input file type is NDJSON. Since NDJSON file can be multidimensional, we normalize the data to two-dimensional data, so that it is suitable to be loaded into a table and this parameter is used as a delimiter for combining columns names if required.
+    example:
+        input JSON:
+
+        .. code:: python
+
+           {"a": {"b": "c"}}
+
+        output table:
+
+        .. list-table::
+           :widths: auto
+
+           * - a_b
+           * - c
+
+    Note - columns a and b are merged to form one col a_b and `_` is used as a delimiter.
+
+    .. literalinclude:: ../../../../example_dags/example_load_file.py
+       :language: python
+       :start-after: [load_file_example_3_start]
+       :end-before: [load_file_example_3_end]
+
+#. **if_exists** - This parameter comes in handy when the table you trying to create already exists. In such a case, we have two options, either replace or append. Which can we accomplish by passing ``if_exists='append'`` or ``if_exists='replace'``.
+
+    .. literalinclude:: ../../../../example_dags/example_load_file.py
+       :language: python
+       :start-after: [load_file_example_4_start]
+       :end-before: [load_file_example_4_end]
+
+#. **output_table** - We can specify the output table to be created by passing in this parameter, which is expected to be an instance of ``astro.sql.table.Table``. Users can specify the schema of tables by passing in the ``columns`` parameter of ``astro.sql.table.Table`` object, which is expected to be a list of the instance of ``sqlalchemy.Column``. If the user doesn't specify the schema the schema is inferred using pandas.
+
+    .. literalinclude:: ../../../../example_dags/example_load_file.py
+       :language: python
+       :start-after: [load_file_example_5_start]
+       :end-before: [load_file_example_5_end]
+
+
+Parameters to use when loading a file to pandas dataframe
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#. ``columns_names_capitalization`` Control col names case of the dataframe generated from the file. The default value is ``original``.
```

Comment:
Since this is also used in the case of Snowflake's table creation (and affects the table creation in any database), perhaps we should not keep it in the dataframe-specific section.

Reply:
I feel that this argument will not work if the user specifies the `table.columns` and `if_exists=replace`. Could you confirm and log a ticket if that is the case?

Reply:
Added `columns_names_capitalization` to database section as well as dataframe, since it has a different meanings in both cases.